packages/medusa/src/api-v2/admin/fulfillment-sets/[id]/service-zones/route.ts

```diff
@@ -0,0 +1,45 @@
+import { createServiceZonesWorkflow } from "@medusajs/core-flows"
+import {
+  ContainerRegistrationKeys,
+  remoteQueryObjectFromString,
+} from "@medusajs/utils"
+import { MedusaRequest, MedusaResponse } from "../../../../../types/routing"
+import { AdminCreateFulfillmentSetServiceZonesSchemaType } from "../../validators"
+
+export const POST = async (
+  req: MedusaRequest<AdminCreateFulfillmentSetServiceZonesSchemaType>,
```

Comment:
Q: do we include schema in all types? I didn't on my side, should it be a convention?

Reply:
I am fine with either. I can remove the `Schema`